src/util/types.tsx

```diff
@@ -2,9 +2,9 @@ export interface AppState {
     list: {
     displayName: string,
     queryName: string
-    }[] | [],
+    }[],
     error: string,
-    favorites: SingleBook[] | []
+    favorites: SingleBook[]
```

Comment:
good to know this! TypeScript is new for all of us, so helpful to learn these little tidbits, like a union is unnecessary here.